### PR TITLE
Switch to GH actions

### PR DESCRIPTION
**Pull request by odow**
Let's see if this works.

### CODE REVIEW

**Comment by mlubin** [comment]
Nice! The other complex part that's not tested here is pushing documentation from documenter.

**Comment by odow** [comment]
Documenter looks like it is okay: https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#GitHub-Actions

**Comment by jd-lara** [comment]
@mlubin we do it this way if it helps 
https://github.com/NREL-SIIP/PowerSystems.jl/blob/master/.github/workflows/docs.yml

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['1.0', '1']  # Test against LTS and current minor release
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        arch: [x64]
+        include:
+          # Also test against Windows 32-bit on LTS.
+          - version: '1.0'
```
Comment (jd-lara) [inline .github/workflows/ci.yml:19]: Is it worth testing against windows 32bit considering it is the second tier in Julia support?

Comment (odow) [reply .github/workflows/ci.yml:19]: This is what we were doing previously, so I kept it. This is where having system information of who is installing each package would be useful...

Comment (mlubin) [reply .github/workflows/ci.yml:19]: 32-bit linux would probably be more useful to test than 32-bit windows, but they provide about the same coverage of the code

**Review by mlubin — APPROVED** [review]

**Comment by mlubin** [comment]
@odow Do you have the access you need to delete the appveyor project?

**Comment by odow** [comment]
> Do you have the access you need to delete the appveyor project?

Yes. I'll merge this now and check it passes on the `master` branch.

**Pull request merged**